src/Services/cdl/CdlHeader.js

```
@@ -12,75 +12,97 @@ class CdlHeader {
-                default:
-                    console.debug(`Unknown SST Node Type:${child.type}`);
-                    break;
+    _getFileFromLogType (logtype) {
+        for (const fileName in this.header.fileTree) {
+            const minLt = this.header.fileTree[fileName].minLt;
+            const maxLt = this.header.fileTree[fileName].maxLt;
+            if (minLt <= logtype && maxLt >= logtype) {
+                return fileName;
             }
-        });
+        }
```

ESLint

[error] The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.

(guard-for-in)